### PR TITLE
Work proof flow

### PR DESCRIPTION
**Pull request by MixailE**
image

### CODE REVIEW

File: src/components/work-flow/WorkFlowCard.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                    : 'Creating your zk proof'}
+                </HeaderText>
+                <BodyText>
+                  {minted
+                    ? proof.result
+                      ? 'You already have a zkBadge verifying your work email! You can choose another account to mint new zkBadge!'
+                      : 'You’ve minted a zkBadge verifying your work email!'
+                    : proof.result
+                    ? 'Mint your zkBadge below to add it to your wallet.'
+                    : 'This may take a second.'}
```
Comment (smyzin) [inline src/components/work-flow/WorkFlowCard.tsx:71]: It looks a little bit complex. Maybe you can set it into a variable?

File: src/components/work-flow/WorkFlowCard.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        {account ? (
+          <Suspense fallback={<LoadingTitle />}>
+            {proof ? (
+              <>
+                <HeaderText extraLeading>
+                  {minted
+                    ? 'Congrats!'
+                    : proof.result
+                    ? 'Ready to mint!'
+                    : 'Creating your zk proof'}
```
Comment (smyzin) [inline src/components/work-flow/WorkFlowCard.tsx:62]: Let's put it into the `headerTitle` variable. What's you think?

File: src/components/work-flow/WorkFlowCard.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    <CardContainer>
+      <Card
+        color="accent"
+        shadow
+        onlyWrap
+        spinner="Work hard, play hard, whistleblow hard"
+      >
+        {account ? (
+          <Suspense fallback={<LoadingTitle />}>
+            {proof ? (
```
Comment (smyzin) [inline src/components/work-flow/WorkFlowCard.tsx:55]: Can you separate it into two independent components?

File: src/components/Announcement.tsx [hunk 4]
```diff
@@ -47,20 +47,16 @@ export default function () {
...
-      <LinkText url="/app">
+      <LinkText url={announcementPage}>
         <AccentText small bold color="text-formal-accent">
           Now introducing zk proof for your work email!{' '}
-          {!onPage
-            ? 'Get started'
-            : account
-            ? 'Get started below.'
-            : 'Connect wallet to get started.'}
+          {account ? 'Get started below.' : 'Connect wallet to get started.'}
```
[the remark this answers is not in the document]
Comment (MixailE) [reply src/components/Announcement.tsx:59]: I'm not sure if this should be in the announcement

**Pull request closed without merging**